### DISCUSSION TITLE
JSON format for observing list

### DISCUSSION THREAD

**Discussion by LasaVecu** [Stellarium/stellarium#2164] [Q&A]
I am trying to convert a simple .txt  list of objects into a .json file that Stellarium would import but  am stumped by two mysterious elements: 1) the line below "observingLists" and 2) the number in the last line.

{
    "observingLists": {
        "{d00f3c1b-df23-4ef8-8e98-d285623eeefc}": {
            "description": "Easily observable clusters",
            "jd": "2459554.492179",

Any hint would be appreciated.

**Comment by gzotti** [comment]
This feature is currently under development. The first item looks like a unique identifier, the jd means Julian Day. While I don't know the application of a date for a complete list, adding dates to observable object entries (upcoming) allows loading things like eclipses or meteor storms.

**Comment by dhdeangelis** [comment]
I see the Observing List feature has now been implemented in master. Thanks!

I wonder where can I find a brief summary of the format required for writing a list that can be imported into Stellarium.

Tanks in advance

**Reply by gzotti** [reply]
This feature is still not fully complete. Format should then be given in the User Guide.
